go/flags/endtoend/vtctld.txt

```diff
@@ -1,6 +1,5 @@
 Usage of vtctld:
       --action_timeout duration                                          time to wait for an action before resorting to force (default 2m0s)
-      --allowed_tablet_types []topodatapb.TabletType                     Specifies the tablet types this vtgate is allowed to route queries to.
       --alsologtostderr                                                  log to standard error as well as files
       --app_idle_timeout duration                                        Idle timeout for app connections (default 1m0s)
       --app_pool_size int                                                Size of the connection pool for app connections (default 40)
```

Comment:
We should be able to remove these two as well.

Reply:
done